pkg/api/ca.go

```diff
@@ -203,7 +203,7 @@ func signingCert(w http.ResponseWriter, req *http.Request) {
 	}
 
 	// Set the SCT and Content-Type headers, and then respond with a 201 Created.
-	w.Header().Add("SCT", string(sctBytes))
+	w.Header().Add("SCT", base64.RawStdEncoding.EncodeToString(sctBytes))
```

Comment:
Since the client uses StdEncoding when decoding, should this use that too?

Reply:
Doh, changed.